exp-player/addon/styles/addon.scss

```diff
@@ -37,6 +37,14 @@
     margin: 5px;
 }
 
+.margin-top-sm {
+    margin-top: 25px;
```

Comment:
Are these definitions of small and large unique to your use case, or are they more generic? How did you choose these numbers? 

Remember, every single exp-frame will see this style name, and when you move your exp-frames to a different repo*, you might want a way to identify which styles need to go with it. Generic names for special-purpose things will make those styles harder to find later.

Aside from general advice, I'm going to leave the final decision on merging to you on this one, since these are pure UI changes specific to ISP. 



* There's no guarantee we'll be moving exp-frames out of exp-addons and into ISP. But it is a change that we are strongly considering as part of the Lookit platform/ phase 2 discussions, and planning ahead a bit will make our lives easier later.